docs/BreakingChanges/024 EncoderParameter ctor is obsolete.md

```diff
@@ -0,0 +1,22 @@
+## 24: EncoderParameter ctor is obsolete
+
+### Scope
+Minor
+
+### Version Introduced
+4.5
+
+### Change Description
+The `EncoderParameter.EncoderParameter(Encoder, Int32, Int32, Int32)` constructor is obsolete now and will introduce build warnings if used.
+
+- [ ] Quirked
+- [x] Build-time break
+- [ ] Source analyzer available
+
+### Recommended Action
+Although the `EncoderParameter.EncoderParameter(Encoder, Int32, Int32, Int32)`  constructor will continue to work, the following constructor should be used instead to avoid the obsolete build warning when re-compiling code with .NET 4.5 tools: [EncoderParameter.EncoderParameter(Encoder, Int32, EncoderParameterValueType, IntPtr)](https://msdn.microsoft.com/en-us/library/hh875096(v=vs.110).aspx).
```

Comment:
Good to know. I removed them because it seemed to work fine, and then I wondered if you had originally escaped them for some reason. If it works in GitHub, we'll leave them for now.